gammapy/makers/map.py

```diff
@@ -340,6 +340,12 @@ def make_meta_table(observation):
         meta_table["OBS_ID"] = [observation.obs_id]
         meta_table["RA_PNT"] = [observation.pointing_radec.icrs.ra.deg] * u.deg
         meta_table["DEC_PNT"] = [observation.pointing_radec.icrs.dec.deg] * u.deg
+        try:
+            meta_table["ZEN_PNT"] = [observation.pointing_zen]
+            meta_table["LIVETIME"] = [observation.observation_live_time_duration]
+        except KeyError:
```

Comment:
It's very non-obvious why this would raise a `KeyError`. You are accessing properties here. One needs to know that these properties in turn do a key lookup in the `meta` dict. 

Also logging a warning about missing optional information is not really good I think.

Reply:
I can make it into an info statement. We generally have warnings in Gammapy, so I stayed with warning. But I agree

Reply:
This is not a warning, it is a log message with level warning. This is an important distinction. Also, if `ZEN_PNT` is not there, you could check for `ALT_PNT` and calculate it.

Reply:
Another possible issue with the implementation is that if the first one is not there then no livetime will be added.
This would require a try-except statement for each entry